src/frequenz/sdk/actor/power_distributing/result.py

```diff
@@ -0,0 +1,111 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Results from PowerDistributingActor."""
+
+from abc import ABC
+from typing import Set
+
+from .request import Request
+
+
+class Result(ABC):
+    """Base class for the power distributor result."""
+
+    def __init__(self, request: Request) -> None:
+        """Create class instance.
+
+        Args:
+            request: Initial request
+        """
+        self.request: Request = request
+
+
+class Success(Result):
+    """Send if setting power for all batteries succeed."""
+
+    def __init__(
+        self, request: Request, succeed_power: int, outside_bound: int
+    ) -> None:
+        """Create class instance.
+
+        Args:
+            request: Initial request
+            succeed_power: How much power out of requested were successfully set.
+            outside_bound: How much power was ignored, because was outside power
+                bound. If Request.adjust_power = False, then this value is always 0.
+        """
+        super().__init__(request)
+        self.succeed_power: int = succeed_power
+        self.outside_bound: int = outside_bound
+
+
+class AnyFail(Result):
```

Comment:
Why is it called `AnyFail`?

Reply:
Example:
5 batteries are charged.
One of them fail ( it won't response or it will response with error).
Then AnyFail error will be send. Because at least one battery failed.

Reply:
Does `PartialFailure` work better?

Reply:
Sounds better, thanks! Done.